python/timing/benchmark.py

```diff
@@ -104,23 +125,27 @@ def benchmark(shapes : List[List[int]], batchsizes: List[int], mem_size: List[in
 
     return report
 
-
-# pylint: disable=redefined-outer-name
+    # pylint: disable=redefined-outer-name
 if __name__ == "__main__":
-
-    #benchmark(shapes : List[List[int]], batchsizes: List[int], mem_size: List[int]) -> pd.DataFrame:
-    #shapes = [[1024, 1024], [10240,10240]]
+    torch.backends.cuda.matmul.allow_tf32 = False
+    controls['triton_allow_tf32'] = False
+    controls['triton_allow_autotune'] = False
+    # benchmark(shapes : List[List[int]], batchsizes: List[int], mem_size: List[int]) -> pd.DataFrame:
+    # shapes = [[1024, 1024], [10240,10240]]
     shapes = [[10240, 10240]]
-    #batch_sizes = [64, 512, 4096, 32768]
-    #batch_sizes = [64, 512, 1024, 4096, 10240]
-    #batch_sizes = [64, 512, 4096]
+    # batch_sizes = [64, 512, 4096, 32768]
+    # batch_sizes = [64, 512, 1024, 4096, 10240]
+    # batch_sizes = [64, 512, 4096]
     batch_sizes = [512]
 
-    #mem_size = [1, 8, 64, 512] + [32768 * 8 ** i for i in range(5)] + [nb_embeddings*embeding_dim]
-    #mem_size = [64, 512] + [32768 * 8 ** i for i in range(5)]
-    mem_size=[1024*1024, 1024*1024*8, 1024*1024*16, 1024*1024*32, 1024*1024*64, 1024*1024*128]
-    #mem_size=[1024*1024, 1024*1024*8]
+    # mem_size = [1, 8, 64, 512] + [32768 * 8 ** i for i in range(5)] + [nb_embeddings*embeding_dim]
+    # mem_size = [64, 512] + [32768 * 8 ** i for i in range(5)]
+    mem_sizes = [1024*1024, 1024*1024*8, 1024*1024 *
+                 16, 1024*1024*32, 1024*1024*64, 1024*1024*128]
+    # mem_size=[1024*1024, 1024*1024*8]
 
-    my_report = benchmark(shapes, batch_sizes, mem_size)
+    # autotune(batch_sizes=batch_sizes, shapes=shapes,
```

Comment:
Uncomment this line if you need performance reporting. It will enable expensive autotuning.